feat: Adding SSL support for online server (#4677)
feast-dev/feast: docs/reference/online-stores/remote.md

online_store:
  path: http://localhost:6566
  type: remote
  ssl_cert_path: /path/to/cert.pem
entity_key_serialization_version: 2
auth:
  type: no_auth
```
{% endcode %}

> `ssl_cert_path` is optional configuration. Path to the public certificate path in case if the online server starts in SSL mode. This may be needed especially if online server started with a self-signed certificate, typically this file ends with .crt, .cer, or .pem.

Reviewer: ```suggestion
`ssl_cert_path` is an optional configuration to the public certificate path when the online server starts in SSL mode. This may be needed if the online server is started with a self-signed certificate, typically this file ends with `*.crt`, `*.cer`, or `*.pem`.
```
